admittance_controller/src/admittance_rule.cpp

```diff
@@ -191,6 +191,10 @@ controller_interface::return_type AdmittanceRule::reset()
   get_pose_of_control_frame_in_base_frame(current_pose_ik_base_frame_);
   feedforward_pose_ik_base_frame_ = current_pose_ik_base_frame_;
 
+  if (open_loop_control_) {
```

Comment:
Can you please leave a comment about the logic here? I don't follow without really digging in to it.

Reply:
added.